src/vue/pages/statistics/StatisticsFilters.vue

```diff
@@ -153,6 +153,16 @@ export default {
         this.$emit(EVENTS.setFiltersAndUpdateList, value)
       },
     },
+    'filters.dateTo' () {
+      if (moment(this.filters.dateTo).isBefore(moment(this.filters.dateFrom))) {
+        this.filters.dateTo = ''
+      }
+    },
+    'filters.dateFrom' () {
+      if (moment(this.filters.dateFrom).isBefore(moment(this.filters.dateTo))) {
```

Comment:
I think it's a bug
If this expression will be true, it will be a normal situation
moment('2010-10-20').isBefore('2010-10-21'); // true
https://momentjs.com/docs/#/query/is-before/

Reply:
fixed